CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## unreleased 2021-02-25
+
+- Bugfix. When space created through api the creator was not added to space as admin. Fixed.
```

Comment:
can you link to actual issue as well. Also if you use `fixes #number` in your commit, it will close the issue when the PR is merged
```suggestion
- Bugfix. When space created through api the creator was not added to space as admin. Fixed.
```
### Fixed
- When space created through api the creator was not added to space as admin. #<issuenumber>

Reply:
Done